modules/virt-adding-vm-to-service-mesh.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-connecting-vm-to-service-mesh.adoc
+
+[id="virt-adding-vm-to-service-mesh_{context}"]
+= Configuring a virtual machine for the service mesh
+
+To add a virtual machine (VM) workload to a service mesh, enable automatic sidecar injection in the VM configuration file by setting the `sidecar.istio.io/inject` annotation to `true`. Then expose your VM as a service to view your application in the mesh.
```

Comment:
WDYT about saying in the mesh dashboard?

Reply:
I don't see any reference to 'mesh dashboard' in the Service Mesh docs, so I am going to leave this as is.

Reply:
ack